packages/shared/src/utils/constants/contract.tsx

```diff
@@ -459,6 +427,38 @@ export const getSupportedContracts = (is_high_low?: boolean) => ({
         name: <Localize i18n_default_text='Put' />,
         position: 'bottom',
     },
+    RUNHIGH: {
```

Comment:
Are you moving the rest of the contracts into this list for the future supported contracts?

Reply:
I'd prefer to keep the scope of the card to ASIAN contracts but no issues from my side 👍🏼